clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -1805,15 +1807,26 @@ private boolean processBackgroundEvents() {
      * execution of the rebalancing logic. The rebalancing logic cannot complete until the
      * {@link ConsumerRebalanceListener} callback is performed.
      *
-     * @param future         Event that contains a {@link CompletableFuture}; it is on this future that the
-     *                       application thread will wait for completion
-     * @param timer          Overall timer that bounds how long to wait for the event to complete
+     * @param future                    Event that contains a {@link CompletableFuture}; it is on this future that the
+     *                                  application thread will wait for completion
+     * @param timer                     Overall timer that bounds how long to wait for the event to complete
+     * @param ignoreErrorEventException When processing background events, some error events are not triggered by current call.
```

Comment:
Should we clearly say what this param is before giving all the context about when it's used? Maybe just saying that is a "_predicate to ignore background errors. Any exceptions found while processing background events that match the predicated won't be propagated_"? something like it. Then probably the explanation about the example of subscribe+invalid would fit better on the caller (ie. on ln 1481) and not in the param description?

Reply:
Thanks for review and suggestion. Updated it.